### PR TITLE
feat: added rust projects

### PR DESCRIPTION
**Pull request by SherbetLemon47**
For #3

### CODE REVIEW

File: docs/rust.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
 # Awesome Open Source - Rust
 
-Coming Soon!
+- [actix-web](https://github.com/actix/actix-web) - Actix Web is a powerful, pragmatic, and extremely fast web framework for Rust.
+- [alacritty](https://github.com/alacritty/alacritty) — A cross-platform, GPU enhanced terminal emulator
+- [Asymmetrik/ngx-deploy-azure](https://github.com/Asymmetrik/ngx-deploy-azure) - Deploy Angular apps to Azure.
```
Comment (stocks29) [inline docs/rust.md]: this repo doesn't exist

File: docs/rust.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+- [bat](https://github.com/sharkdp/bat) - A cat(1) clone with syntax highlighting and Git integration.
+- [chrono](https://github.com/chronotope/chrono) - Date and time library for Rust.
+- [clap](https://github.com/clap-rs/clap) - A simple to use, full-featured Command Line Argument Parser for Rust.
+- [diesel](https://github.com/diesel-rs/diesel) - A safe, extensible ORM and Query Builder for Rust.
+- [dioxus](https://github.com/DioxusLabs/dioxus) - Elegant React-like library for building user interfaces for desktop, web, mobile, and more.
+- [fake-rs](https://github.com/cksac/fake-rs) - A library for generating fake data.
+- [Fuchsia OS](https://fuchsia.dev/) - A new open source operating system created at Google.
+- [glium](https://github.com/glium/glium) - Safe OpenGL wrapper for the Rust language.
+- [google/benchmark](https://github.com/google/benchmark) - A microbenchmark support library.
```
Comment (stocks29) [inline docs/rust.md]: this is a c++ package

**Review by stocks29 — CHANGES_REQUESTED** [review]
@SherbetLemon47 thanks for submitting this PR. there are a few minor tweaks needed. please see the inline comments.

File: docs/rust.md (hunk 1 above)
Comment (SherbetLemon47) [reply docs/rust.md]: Removed. Added 2 more projects in favor of removals.

File: docs/rust.md (hunk 2 above)
Comment (SherbetLemon47) [reply docs/rust.md]: Removed! Read about it in a article measuring rust speeds so assumed it was in rust. My bad.

Comment (stocks29) [reply docs/rust.md]: no worries, thanks sir!

**Review by stocks29 — APPROVED** [review]

**Pull request merged**